src/components/Address.gql

```diff
@@ -1,4 +1,4 @@
-fragment BaseAddress on Address {
+fragment AddressFields on Address {
```

Comment:
Trying to give it a more generic name. "Base" sounded as there could be an extended version.